src/components/Box/Box.md

```diff
@@ -7,5 +7,40 @@ Add here any notes on how to use the component, or delete this section.
 Basic usage:
 
 ```js { "props": { "data-example": "basic" } }
-<Box />
+<Box>This is a basic box</Box>
 ```
+
+Different border types:
+```js { "props": { "data-example": "borderType" } }
```

Comment:
I'd leave a new line between the description and the code block, to let it breather a bit.